### DISCUSSION TITLE
[@types/eslint] no types for 8.57.0

### DISCUSSION THREAD

**Discussion by cbejensen** [DefinitelyTyped/DefinitelyTyped#71207] [Issues with a @types package] [answered]
We are using `eslint@8.57.0` but the latest `@types/eslint` pkg I see is `8.56.12`. Am I missing something? I'd like to use `loadESLint` which was introduced in `8.57.0` but I'm getting a type error.

[a comment by an automated account is not shown]

**Comment by jakebailey** [comment, marked as answer]
This package was deleted from DT as eslint ships types themselves. I guess the package never got v8.57, so if we wanted to handle this, we'd have to restore the package from git and then delete it again after. Is it worth it, if eslint v8 is deprecated?

**Comment by cbejensen** [comment]
No worries, I’ll just throw a `@ts-expect-error` on it.
